utils/decorator.py

```diff
@@ -0,0 +1,19 @@
+import json
+import bcrypt
+import jwt
+
+from user.models import User
+
+from my_settings import SECRET_KEY, ALGORITHM
+
+def authorization (func):
+    def wrapper (self, request, *args, **kwrgs):
+        try:
+            token         = request.headers['Authorization']
+            decoded_token = jwt.decode(token, SECRET_KEY, ALGORITHM)
+            request.user  = User.objects.get(id=decoded_token["user_id"])
+            return func(self, request, *args, **kwargs)
+            
+        except jwt.DecodeError:
+            return JsonResponse({'message': 'Not exact token'}, status=401)
```

Comment:
```suggestion
            return JsonResponse({'message': 'INVALID_TOKEN'}, status=401)
```
INVALID_TOKEN이 더 어울릴 것 같습니다.